src/backend/cdb/motion/test/Makefile

```diff
@@ -0,0 +1,9 @@
+subdir=src/backend/cdb/motion
+top_builddir=../../../../..
+include $(top_builddir)/src/Makefile.global
+
+TARGETS=cdbsenddummypacket
```

Comment:
How do these tests run in CI?

Reply:
They run in the "Build for xxx" job. But there is a caveat: all our test machines have IPv6 enabled, so [the IPv4 branch](https://github.com/arenadata/gpdb/pull/679/files#diff-b9a680f7851e2128e23bfbde09f08cc0a5c3ab94858cdfb0ec283ab9f6668313R226-R233) inside the `test_send_dummy_packet_default_wildcard()` will not be covered by our CI.

Reply:
> They run in the "Build for xxx" job.

OK, I see

Reply:
> all our test machines have IPv6 enabled, so [the IPv4 branch](https://github.com/arenadata/gpdb/pull/679/files#diff-b9a680f7851e2128e23bfbde09f08cc0a5c3ab94858cdfb0ec283ab9f6668313R226-R233) inside the `test_send_dummy_packet_default_wildcard()` will not be covered by our CI.

I tested it manually